src/test/java/gov/healthit/chpl/aqa/pageObjects/ListingDetailsPage.java

```diff
@@ -9,30 +9,30 @@
 /**
  * Class ListingDetailsPage definition.
  */
-public final class ListingDetailsPage {
+public final class ListingDetailsPage extends BasePage {
 
     private static final int CRITERIA_COLUMN = 3;
 
     private ListingDetailsPage() {}
-    private static WebElement element = null;
+
     /**
      * Returns element that holds Accessibility Standard Process Text.
      * @param driver WebDriver
      * @return Text element
      */
     public static WebElement accessibilityStandardText(final WebDriver driver) {
-        element = driver.findElement(By.xpath("//*[@id=\"panel-certification-criteria\"]/div[54]/div/div/div/table/tbody/tr[3]/td[2]/ul/li[1]"));
-        return element;
+        return driver.findElement(By.xpath("//*[@id=\"panel-certification-criteria\"]/div[54]/div/div/div/table/tbody/tr[3]/td[2]/ul/li[1]"));
```

Comment:
Is this targeting a specific criteria in the list of them? Will it work on a listing that doesn't attest to that criteria? Or maybe on one that only attests to 3 or 4 criteria? I see it's looking for the 54th something...

Reply:
This element was not used in the CQM test. I'll review and update it as necessary in next PR. I removed all 'element =' and added return to element definition to all elements in this page object file, that's why this shows in changes.

Reply:
Ahh, I'd missed that it wasn't really a change in functionality